src/oci.c

```diff
@@ -229,6 +229,11 @@ cc_oci_get_config_and_state (gchar **config_file,
 			(*state)->procsock_path,
```

Comment:
nit: ' , ' whitespace in commit message, and this one is 'tests' and the previous one is 'test' :-)